t_health_monitoring/test_health_monitor.py

```diff
@@ -173,12 +174,13 @@ def wait_for_server(self, srv):
         srv.wait_for_connections()
 
     def run_curl(self, n=1):
-        res = []
+        res = defaultdict(lambda: 0)
```

Comment:
I would recommend to simplify the implementation, but it is up to you.
```suggestion
        res = defaultdict(int)
```